emilia/modules/welcome.py

```diff
@@ -370,6 +370,7 @@ def check_bot_button(update, context):
 			return
 		# Unmute user
 		context.bot.restrict_chat_member(chat.id, user.id, permissions=ChatPermissions(can_send_messages=True, can_send_media_messages=True, can_send_other_messages=True, can_add_web_page_previews=True))
+		bot.deleteMessage(chat.id, message.message_id)
```

Comment:
This only affect 'all restirect by welcsec' member verify, also it was bad method to unmute member since someone can bypass admin mute. Ik left welcome message after verified is spammed chat, but it still needed for some reasons (like custom button url, rules, etc)

Reply:
Yes,you r right👌keep it up